website/docs/components/code-block/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,96 @@
+## Usage
+
+### When to use
+
+- When displaying code examples and longer snippets of code that benefit from syntax highlighting.
+
+### When not to use
+
+- As a full-blown code editor.
+
+### When to use a CodeBlock vs a Copy Snippet
```

Comment:
[suggestion] This section may benefit from some visual do/don't examples.

Reply:
Ok, I've added a couple of examples. I think they're probably representative enough, but I'm not 100% sold on them. WDYT?

Reply:
Personally, I don't think adding do/don't examples is adding much clarity. There is an intrinsic ambiguity between the usage of two components (_"There is some overlap in the copying functionality of the Copy Snippet and the CodeBlock"_) and for me the visual examples add to the confusion, instead of providing clarity. We should trust the consumers to make the right choice, in their context, since we can't ourselves pinpoint to specific criteria on how to choose which.

Reply:
Agreed, I'll remove them.